pkg/config/render_config.go

```diff
@@ -41,6 +41,7 @@ type context struct {
 	TraceAgent        bool
 	ClusterChecks     bool
 	CloudFoundryBBS   bool
+	SecurityModule    bool
```

Comment:
this is false by default, so won't be included unless you set it to true for some agent build types in `mkContext`

Reply:
Good catch. Fixed